app/src/utils/get-default-values-from-fields.ts

```diff
@@ -0,0 +1,21 @@
+import { Field } from '@directus/shared/types';
+import { Ref, unref, computed, ComputedRef } from 'vue';
+
+export function getDefaultValuesFromFields(fields: Field[] | Ref<Field[]>): ComputedRef<Record<string, any>> {
+	return computed(() => {
+		return unref(fields).reduce(function (acc, field) {
+			if (
+				field.schema?.default_value !== undefined &&
+				// Ignore autoincremented integer PK field
+				!(
+					field.schema.is_primary_key &&
+					field.schema.data_type === 'integer' &&
+					typeof field.schema.default_value === 'string'
+				)
+			) {
+				acc[field.field] = field.schema.default_value;
```

Comment:
wondered about the original redundant optional chaining of `field.schema?.default_value` in v-form when looking at it again, but you're already one step ahead 😆